### ISSUE TITLE
[FP]: Eclipse IDE vulnerability reported for Jakarta Jax-RS API

### ISSUE THREAD

**Issue by snemets** [dependency-check/DependencyCheck#4220]
### Package URl

pkg:maven/jakarta.ws.rs/jakarta.ws.rs-api@2.1.6

### CPE

cpe:2.3:a:eclipse:eclipse_ide:2.1.6:*:*:*:*:*:*:* 

### CVE

CVE-2008-7271, CVE-2010-4647 

### ODC Integration

{"label"=>"Maven Plugin"}

### ODC Version

7.0.0

**Comment by Janpopan** [comment]
same #4203

**Comment by Janpopan** [comment]
Duplicate of #4129?

so solved by

https://github.com/jeremylong/DependencyCheck/pull/4159

**Comment by Janpopan** [comment]
should solved with the new released 7.0.1

**Comment by pappasn-sd** [comment]
It's not solved I still get it flagged, even in v7.0.3 (gradle plugin).

**Comment by aikebah** [comment]
@pappasn-sd 

> It's not solved I still get it flagged, even in v7.0.3 (gradle plugin).

Not this library.... passed both 7.0.1 and 7.0.3 analysis according to my analysis on the following build.gradle

```
buildscript {
    repositories {
        mavenLocal()
        maven {
            url "https://plugins.gradle.org/m2/"
        }
        mavenCentral()
    }
    dependencies {
        classpath "org.owasp:dependency-check-gradle:7.0.3"
    }
}

plugins {
    id 'java-library'
}

repositories {
    mavenLocal()
    mavenCentral()
}

dependencies {
    implementation ( [
        'jakarta.ws.rs:jakarta.ws.rs-api:2.1.6'
    ] )
}

apply plugin: "org.owasp.dependencycheck"
```
